consensus/checks.go

```diff
@@ -63,7 +63,7 @@ func (consensus *Consensus) isRightBlockNumAndViewID(recvMsg *FBFTMessage,
 			Uint64("MsgViewID", recvMsg.ViewID).
 			Uint64("MsgBlockNum", recvMsg.BlockNum).
 			Uint64("blockNum", consensus.blockNum).
-			Str("ValidatorPubKey", recvMsg.SenderPubkey.Bytes.Hex()).
+			Interface("ValidatorPubKey", recvMsg.SenderPubkeys).
```

Comment:
This will give us non-readable hex representation. Is it better to implement a `ToString` method?

Reply:
Changed to log the whole recvMsg which has a better String() func.